server/server.js

```diff
@@ -4,11 +4,14 @@ const express = require('express')
 const server = express()
 
 const starterRoute = require('./routes/starter')
-const assessmentsRoute = require ('./routes/assessmentsRoute')
+const cohortRoutes = require('./routes/cohortRoutes')
```

Comment:
Why is `cohortRoutes` being added? This is unrelated to issue #11 .

Reply:
Hi, Bryce, that's part of conflict resolving when I pull from dev branch, it was added on.